server/events/index.js

```diff
@@ -74,7 +74,16 @@ class RoomEventBridge extends EventEmitter {
     const clientSocket = this._getClientSocketInRoom(
       this._userSocketIdMap[userId]
     );
+    let otherPlayers = [];
+    for (const [key, value] of Object.entries(this._io.sockets.connected)) {
+      if (key !== this._userSocketIdMap[userId]) {
+        otherPlayers = [...otherPlayers, value];
+      }
+    }
+
     if (clientSocket) clientSocket.emit('GE_NEW_WORD', word);
+    // Emit hint word event for others
+    otherPlayers.forEach(playerSocker => playerSocker.emit('GE_NEW_HINT_WORD', word));
```

Comment:
Two things I thought would improve the overall experience and loopholes 
1. How about sending the hint word after certain time has elapsed in the round from the server? I mean can we not send it immediately at the start of the round?
2. Instead of sending the whole word as the hint word, I think we should compute the masked word (hint) from the server and then send it to clients. This will help clients from spoofing others.

Reply:
Even I thought of this. Will do it. Anything else except for this? @mahadevans87